cli/translate.py

```diff
@@ -0,0 +1,88 @@
+import sys
+from typing import Any, Dict, List
+
+from datastore.reader.core import FilterRequest, Reader
+from datastore.reader.services import register_services as register_reader_services
+from datastore.shared.di import injector
+from datastore.shared.util import FilterOperator, fqid_from_collection_and_id
+from datastore.writer.core import RequestUpdateEvent, Writer, WriteRequest
+from datastore.writer.services import register_services as register_writer_services
+
+from openslides_backend.locale.translator import Translator
+
+collection_to_fields_map = {
+    "group": ["name"],
+    "motion_workflow": ["name"],
+    "motion_state": ["name", "recommendation_label"],
+    "projector_countdown": ["name"],
+    "projector": ["name"],
+    "meeting": [
+        "welcome_title",
+        "welcome_text",
+        "name",
+        "description",
+        "users_email_subject",
+        "users_email_body",
+        "assignments_export_title",
+    ],
+}
+
+
+def read_collection(collection: str, fields: List[str]) -> Any:
+    reader: Reader = injector.get(Reader)
+    with reader.get_database_context():
+        response = reader.filter(
+            FilterRequest(
+                collection,
+                FilterOperator("meta_deleted", "=", False),
+                ["id", *fields],
+            )
+        )
```

Comment:
Don't use a filter, especially not with so many results. Fetch the relations in order (`organization`->`meeting`->meeting-related model`) and apply the translations in each step. Only use `get` and `get_many` with `deleted_models_behaviour=NO_DELETED`.

Reply:
I use an ordered dictionary to set the relation order, and reorder the relations in collection_to_fields_map.

Reply:
I use get_all, that is what I want.

Reply:
ok, but then you can remove the `OrderedDict` again